libraries/core_libs/consensus/src/dag/sortition_params_manager.cpp

```diff
@@ -136,17 +138,23 @@ int32_t efficiencyToChange(uint16_t efficiency, uint16_t goal_efficiency) {
   return UINT16_MAX / 20;
 }
 
-int32_t SortitionParamsManager::getNewUpperRange(uint16_t efficiency) const {
-  assert(params_changes_.size() > 0);
-  const auto last_threshold_upper = params_changes_[params_changes_.size() - 1].vrf_params.threshold_upper;
-  if (efficiency >= config_.dag_efficiency_targets.first && efficiency <= config_.dag_efficiency_targets.second) {
-    return last_threshold_upper;
+int32_t getClosestThreshold(const efficiencies_map& efficiencies, uint16_t target, bool is_over_target) {
```

Comment:
int32_t getClosestThreshold(const efficiencies_map& efficiencies, uint16_t target, bool is_over_target) **const**

Reply:
function(not a method) couldn't be const